internal/service/account/region.go

```diff
@@ -78,7 +77,10 @@ func resourceRegionUpdate(ctx context.Context, d *schema.ResourceData, meta inte
 	accountID := ""
 	if v, ok := d.GetOk(names.AttrAccountID); ok {
 		accountID = v.(string)
-		id = errs.Must(flex.FlattenResourceId([]string{accountID, region}, regionResourceIDPartCount, false))
+		id, err := flex.FlattenResourceId([]string{accountID, region}, regionResourceIDPartCount, false)
+		if err != nil {
+			return sdkdiag.AppendErrorf(diags, "enabling Account Region (%s): %s", id, err)
```

Comment:
I guess `id` will always be `""` here (and in other resources) as `err != nil`.
May be simpler to just `return sdkdiag.AppendFromErr(diags, err)`? These cases occur rarely and Terraform core may provide enough context (operation and resource name).

Reply:
Fixed!